playwright_test/tests/chatOptions.spec.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * chatOptions.spec.js
+ * playwright_test/tests
+ * Created by Archit on 23/12/2011. 
+ */
+
+import { test, expect } from '@playwright/test';
+import { widgetLocators, locators } from '../locaterPackage/kmLocators';
+import { url , app_id } from '../utils/kmSecret';
+import { script } from '../utils/kmScript';
+
+let browser;
+let page;
```

Comment:
no need to declare variable for the page and browser 
you can directly pass it in function
![image](https://user-images.githubusercontent.com/113094394/210330395-63151c35-1837-4662-b5ec-4fb18866baa1.png)

Reply:
ya, I forget to delete the browser part that I will delete in the upcoming PR also if I delete the page test will start falling.